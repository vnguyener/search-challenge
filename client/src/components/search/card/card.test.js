import React from 'react';
import { Provider } from 'react-redux';
import { configure } from 'enzyme';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import Adapter from 'enzyme-adapter-react-16';
import SearchCard from '.';

configure({ adapter: new Adapter() });
const mockStore = configureMockStore([thunk]);

describe('search card component', () => {
  it('renders correctly', () => {
    const store = mockStore();

    const wrapper = renderer
      .create(
        <Provider store={store}>
          <SearchCard />
        </Provider>
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
