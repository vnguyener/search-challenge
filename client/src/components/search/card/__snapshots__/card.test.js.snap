// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search card component renders correctly 1`] = `
<div
  className="search-card"
>
  <div
    className="box-shadow"
  >
    <div
      className="avatar clickable"
      onClick={[Function]}
    >
      <img
        alt="potential date"
        src=""
      />
      <div
        style={
          Object {
            "borderRadius": "inherit",
            "bottom": "0",
            "overflow": "hidden",
            "position": "absolute",
            "width": "100%",
          }
        }
      >
        <div
          style={
            Object {
              "alignItems": "flex-end",
              "display": "flex",
              "justifyContent": "space-between",
              "margin": 8,
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "color": "white",
                "display": "flex",
                "flexDirection": "column",
                "width": "100%",
              }
            }
          >
            <h6
              style={
                Object {
                  "fontSize": "16px ",
                }
              }
            >
              <div
                style={
                  Object {
                    "alignItems": "center",
                    "display": "flex",
                    "marginBottom": "4px",
                  }
                }
              >
                
              </div>
            </h6>
            <div
              style={
                Object {
                  "alignItems": "baseline",
                  "display": "flex",
                  "justifyContent": "space-between",
                }
              }
            >
              <div
                style={
                  Object {
                    "alignItems": "center",
                    "display": "flex",
                    "marginBottom": "4px",
                  }
                }
              >
                <span>
                  99
                </span>
              </div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "height": "15px",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
